builtin/providers/cloudflare/resource_cloudflare_record.go

```diff
@@ -44,15 +47,25 @@ func resourceCloudFlareRecord() *schema.Resource {
 			},
 
 			"ttl": &schema.Schema{
-				Type:     schema.TypeString,
+				Type:     schema.TypeInt,
 				Optional: true,
 				Computed: true,
 			},
 
 			"priority": &schema.Schema{
-				Type:     schema.TypeString,
+				Type:     schema.TypeInt,
+				Optional: true,
+			},
+
+			"proxied": &schema.Schema{
+				Type:     schema.TypeBool,
 				Optional: true,
```

Comment:
Generally for bools we just reflect the upstream default w/ `Default: false`, then below instead of checking `GetOk` you can simply set `d.Get("proxied").(bool)` directly.